src/components/NavBar/Navbar.js

```diff
@@ -72,6 +75,43 @@ function Navbar() {
               )}
             </ul>
           </div>
+          <div className="md:hidden">
+            <Dropdown label="" inline>
+              {currentUser ? (
+                <Dropdown.Header onClick={() => navigate('/profile')}>

```

Comment:
could you add ( 'cursor-pointer') class so when hovered the cursor visibility changes.

Reply:
ur right I didn't remember that XD